diesel/src/serialize.rs

```diff
@@ -186,7 +186,8 @@ where
 /// database, you should use `i32::to_sql(x, out)` instead of writing to `out`
 /// yourself.)
 ///
-/// Any types which implement this trait should also [`#[derive(AsExpression)]`].
+/// Any types which implement this trait should also
+/// [`#[derive(AsExpression)]`](crate::AsExpression).
```

Comment:
```suggestion
/// [`#[derive(AsExpression)]`](derive@crate::AsExpression).
```

Just to be sure that does not "break" again